.eslintrc.js

```diff
@@ -122,7 +122,6 @@ module.exports = {
     'react/no-deprecated': 2,
     'react/no-direct-mutation-state': 2,
     'react/sort-prop-types': [1, { callbacksLast: true, requiredFirst: true }],
-    'react/jsx-no-bind': [1, { ignoreRefs: true }],
```

Comment:
@cohnjesse I think you want to change these to a `2` instead of removing completely. I'm pretty sure they're default rules in the airbnb base

Reply:
@timwright12 Shouldn't these be a `0` not a `2`? Doesn't a `2` turn them from a warning to an error?

Reply:
yeah you're right, it's `0`

Reply:
@timwright12 done